crypto/crypto/src/hash.rs

```diff
@@ -457,6 +463,16 @@ pub struct DefaultHasher {
 }
 
 impl DefaultHasher {
+    #[doc(hidden)]
+    pub fn prefixed_hash(buffer: &[u8]) -> [u8; 32] {
```

Comment:
FWIW I preferred a name like `make_seed` (because we are not returning a `HashValue` neither is the returned value useful in any other way).

Reply:
The fact that this does not return a `HashValue` is inferrable from the type, but I'll add a docstring.